_posts/205-05-05-glenntatum-simpler-grants.md

```diff
@@ -0,0 +1,199 @@
+into an area where I had just enough experience to advance my skills was important. Contributing to grants looked optimal, as it fit my prior experience
+and my interests in public service and developing software for the greater good of society.
+
+### Getting Involved
+
+I first became involved with the community through the simpler.grants.gov Spring 2025 Hackathon.
+Throughout the 3 weeks in which the hackathon was hosted, I could directly speak with the developers at "Nava," the contractor
+for the simpler.grants.gov project. I attended 3 of their mentorship meetings to seek advice on development and mentorship for working
+with the codebase.
+
+In terms of onboarding, it was an effortless experience. For running the stack locally
+a `Makefile` was provided to run the backend containers with `docker compose`. Although the backend was practically eating up half 
+of my laptops RAM, being that it ran several huge services in combination with a development NextJS server I hosted locally, I faced
+no issues with dependencies, and since everything was containerized from the start with volumes binded to my local environment, having
+features like hot reloading, and being able to easily teardown and build up the application instantly was a huge time saver. Since then, 
+I have incorporated docker compose into more of my project due to the tremendous gains in efficiency you can get with using the utility.
+```Makefile
+# Makefile Run Command, many different services started simultaneously
+init-db init-Opensearch init-localstack init-mock-soap-services
+```
+
+For documentation, everything was stored inside the GitHub repository. I wish more repositories would do this and not have esoteric build systems and formatting for documentation, as then you can have a copy of everything locally. It was verbose. For example, here is documentation on the `api`. 
+```
+├── api
+│   ├── api-details.md
+│   ├── API-versioning.md
+│   ├── authentication.md
+│   ├── database
+│   │   ├── database-access-management.md
+│   │   ├── database-local-usage.md
+│   │   ├── database-management.md
+│   │   ├── database-testing.md
+│   │   └── erds
+│   ├── development.md
+│   ├── error-handling.md
+│   ├── feature-flags.md
+│   ├── formatting-and-linting.md
+│   ├── images
+│   │   ├── health check-response.png
+│   │   ├── local-db-cli.png
+│   │   ├── local-db-connection.png
+│   │   ├── swagger-auth.png
+│   │   └── swagger-ui.png
+│   ├── lookup-values.md
+│   ├── monitoring-and-observability
+│   │   ├── human-readable-logs.png
+│   │   ├── logging-configuration.md
+│   │   └── logging-conventions.md
+│   ├── package-dependency-management.md
+│   ├── README.md
+│   ├── technical-overview.md
+│   └── writing-tests.md
+```
+If you have any questions, the documentation could answer them. When I spoke to engineers at Nava, they said that most of this code was bootstrapped from an open-source template they have on their GitHub.
+
+![nava](glenn-nava.png)
```

Comment:
This image isnt loading, and neither are most of your other ones. Can you move them to the `assets/images` directory and refer to them from there? (the sample post has examples of this)